### PR TITLE
Specifying PSL cache location in trustymail

### PR DESCRIPTION
**Pull request by jsf9k**
Forcing `trustymail` to use the same PSL cache as the rest of `domain-scan` instead of its own.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -125,15 +130,15 @@ RUN npm install --global phantomas phantomjs-prebuilt \
 ###
 RUN pip install --upgrade \
     git+https://github.com/dhs-ncats/pshtt.git@develop \
-    git+https://github.com/dhs-ncats/trustymail.git@develop
+    git+https://github.com/dhs-ncats/trustymail.git@bugfix/problem_with_psl_caching
```
Comment (konklone) [inline Dockerfile]: It looks like this was merged into `develop` already: https://github.com/dhs-ncats/trustymail/pull/72

Should this line get reverted back to `develop`?

Comment (jsf9k) [reply Dockerfile]: Yes, that wasn't intended to be committed.  Thanks for catching that!

Comment (jsf9k) [reply Dockerfile]: @konklone, it looks like you'll have to un-request changes before I can merge.

**Comment by konklone** [comment]
Ah yeah, I set the `master` branch as protected. If anything comes across as too restrictive, very happy to discuss!

**Comment by jsf9k** [comment]
No, I think it's good to do things this way.

**Pull request merged**